### DISCUSSION TITLE
Create a git-hub team in the sociomantic-tsunami org

### DISCUSSION THREAD

**Discussion by llucax** [sociomantic-tsunami/git-hub#307] [Ideas]
Hi @sociomantic-tsunami/devinfra-team!

Currently you are the ones I usually ask for reviews for this repo, but the name is not great and not obvious for other people.

What do you think about creating a specific `git-hub` team?

If we do so, please let me know who wants to be in it and I will create it. @jwilk this goes for you too, as you have been contributing for years now and the main contributor outside the ex-Sociomantic sphere (and I think you had write access in the past, it must have been lost when the company when this got started ceased to exist and this became a 100% voluntary open source project :).

**Comment by ibuclaw** [comment]
I have no issues with this, go do whatever you see fit really.

**Reply by llucax** [reply]
I would do so only if anybody but me wants to join, otherwise I will keep nagging the @sociomantic-tsunami/devinfra-team :smiling_imp:

**Reply by Geod24** [reply]
Who's taking one for the team ? :D

**Reply by ibuclaw** [reply]
@daniel-zullo? :-)

**Reply by daniel-zullo** [reply]
yes, please add me in if you create a new team

**Reply by llucax** [reply]
Ok, let the negotiation begin. I take the Zullo, Zullos are great. But I need more, as Zullos are a bit busy lately they tend to come with some serious latency. I'll create the team if I can take a Buclaw too, they seem be low latency, at least for trivial PRs, so it can become handy...
